### PR TITLE
feat: Removed cartridge avatar & Added crown to Skull icon

### PR DESCRIPTION
**Pull request by akhercha**
#### Related issues
* Add a 'crown' variant to the skull thumbnails -  [#194](https://github.com/cartridge-gg/rollyourown/issues/194)
* Remove Cartridge avatar from list of possible player thumbnail - [#193](https://github.com/cartridge-gg/rollyourown/issues/193) 

___

## Added
* Crown variant to the Skull icon

## Removed
* Cartridge avatar

image

## Comments

Happy to change anything, lemme know!

### CODE REVIEW

File: web/src/components/icons/Skull.tsx
Comment (notV4l) [inline web/src/components/icons/Skull.tsx:1]: fill-rule / clip-rule should be fillRule / clipRule
do we need to duplicate path x4 ? 
you can use { condition && <path ... /> } to show/hide some part of the path 
and fill={condition ? "green" : "yellow"}

Comment (akhercha) [reply web/src/components/icons/Skull.tsx:1]: Ah yeah indeed, was not sure at all how to work properly with those SVGs. 😅
Updated!

For this code:
```html
            <Text>HALL OF FAME</Text>
            <Skull color="yellow"></Skull>
            <Skull color="green"></Skull>
            <Skull color="green" hasCrown={true}></Skull>
            <Skull color="yellow" hasCrown={true}></Skull>
```

I have this (same as before then):
image

File: web/src/components/icons/Skull.tsx [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
-export const Skull = (props: IconProps) => {
+export const Skull = ({
+  hasCrown,
+  color,
+  ...props
+}: IconProps & AvatarPathProps) => {
   return (
     <Icon viewBox="0 0 24 24" {...props}>
-      <path d="M19.0001 7.99998V6.99987H18.0001V5.99994H17.0002V5H6.99998V5.99994H6.00005V6.99987H5.00011V7.99981L4 7.99998V12.9997H4.99994V14.9995H5.99987V15.9995H6.99981V18.9993H8.99969V16.9994H10.9996V18.9993H12.9994V16.9994H14.9993V18.9993H16.9992V15.9995H17.9991V14.9995H18.9991V12.9997H19.999V9.99985L20 8.99992V7.99998L19.0001 7.99998ZM17.0002 12.9997H15.0003V11.9997H14.0004V9.99986H17.0002V12.9997ZM11.0002 14.9997V12.9998H13.0001V14.9997H11.0002ZM7.00015 11.9997V9.99986H9.99997V11.9997H9.0002V12.9997H7.00015V11.9997Z" />
+      <svg width="24" height="24" viewBox="0 0 36 36" fill="none">
```
[the remark this answers is not in the document]
Comment (notV4l) [reply web/src/components/icons/Skull.tsx:11]: you can use <> </> to wrap Icon content if it raise errors

**Review by notV4l — CHANGES_REQUESTED** [review]

File: web/src/components/icons/Skull.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (akhercha) [reply web/src/components/icons/Skull.tsx]: Updated 💀; still gives the same output:

image

**Review by notV4l — APPROVED** [review]

**Pull request merged**